### ISSUE TITLE
Update color of filename when buffer changes.

### ISSUE THREAD

**Issue by kadecole** [atom/tree-view#706]
I would like to be able to update the color of the filename in the tree-view when the buffer for that file has changed. Currently you must save the file for the included git module to update the color. I would like to have the name of the filename change when the buffer is modified, similar to the way currently the X icon changes to a blue circle in the buffer tab.

**Comment by Zireael07** [comment]
Hmm, my two cents: make "changed buffer" color different to the git colors. Methinks blue?

**Comment by kadecole** [comment]
I agree. A nice blue color for the text of the filename in the tree-view.
